src/main/java/BlueTurtle/TSE/JSONFormatter.java

```diff
@@ -35,58 +34,28 @@
 	 */
 	public void format() throws IOException {
 				
-		parseCheckStyleXML(new File(JavaController.getCheckStyleOutputFile()));
+		parseFile(new CheckStyleXMLParser(), JavaController.getCheckStyleOutputFile());
 
-		parsePMDXML(new File(JavaController.getPmdOutputFile()));
+		parseFile(new PMDXMLParser(), JavaController.getPmdOutputFile());
 
-		parseFindBugsXML(new File(JavaController.getFindBugsOutputFile()));
+		parseFile(new FindBugsXMLParser(), JavaController.getFindBugsOutputFile());
 		
 		writeJSON();
 	}
 	
 	/**
-	 * Parse CheckStyle output and produce list of warnings.
+	 * Parse ASAT output and produce list of warnings.
 	 * 
 	 * @return List of warnings.
 	 * @throws IOException
 	 *             File not found.
 	 */
-	private void parseCheckStyleXML(File file) throws IOException {
+	private void parseFile(XMLParser xmlParser, String filePath) throws IOException {
+		File file = new File(filePath);
```

Comment:
If the `file` variable is going to be used once, then just change the condition to: 

```java
if (new File(filePath).exists()) 
```